watchtower/wtclient/backup_task.go

```diff
@@ -141,7 +141,14 @@ func (t *backupTask) bindSession(session *wtdb.ClientSessionBody) error {
 	// Next, add the contribution from the inputs that are present on this
 	// breach transaction.
 	if t.toLocalInput != nil {
-		weightEstimate.AddWitnessInput(input.ToLocalPenaltyWitnessSize)
+		// An older ToLocalPenaltyWitnessSize constant used to
```

Comment:
Yeah this seems easier in the short-term than creating a new blob type all together. This way all the existing towers can continue to work with lnd 0.10 and beyond.

Reply:
yeah working plan is to deploy a small batch that sets a new feature bit during init, and if both parties support, also set a new bit in the blob type that indicates the server should use the updated weights. should be doable for 0.11 and possibly 0.10.1